Examples/Google/SpeechToText/Sources/SpeechService.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  SpeechService.swift
+//  SpeechToText-gRPC-iOS
+//
+//  Created by Prickett, Jacob (J.A.) on 4/13/20.
+//  Copyright © 2020 Prickett, Jacob (J.A.). All rights reserved.
+//
+
+import GRPC
+
+final class SpeechService {
+
+    // Generated SpeechClient for making calls
+    private var client: Google_Cloud_Speech_V1_SpeechClient
+
+    // Stream that is returned from `streamingRecognize` call
+    private var call: BidirectionalStreamingCall<Google_Cloud_Speech_V1_StreamingRecognizeRequest, Google_Cloud_Speech_V1_StreamingRecognizeResponse>?
+
+    // Track if we are streaming or not
+    private var isStreaming: Bool = false
+
+    init() {
+
+        // Specify call options to be used for gRPC calls
+        var callOptions = CallOptions()
```

Comment:
nit: let's create the call options _after_ creating the connection.

Also we can use the expressible by dictionary literal initializer for custom metadata: 

```
let callOptions = CallOptions(customMetadata: [
  "x-goog-api-key": Constants.apiKey,
  "x-ios-bundle-identifier": ...
])
```